auth/auth.py

```diff
@@ -0,0 +1,72 @@
+from flask import (
+    Blueprint,
+    flash,
+    make_response,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from auth.forms import RegisterForm, UserLoginForm
+from .models import User, db
+
+auth = Blueprint('auth', __name__, template_folder='templates')
+
+
+@auth.route('/register', methods=['GET'])
+def register():
+    form = RegisterForm()
+    return render_template('pages/register.html', form=form)
+
+
+@auth.route('/register', methods=['POST'])
+def register_post():
+    form = RegisterForm(request.form)
+    if form.validate():
+        name = form.name.data
+        email = form.email.data
+        password = generate_password_hash(form.password.data)
+        user = User(name=name, email=email, password=password)
+        db.session.add(user)
+        db.session.commit()
+        flash('User created successfully!')
+        return redirect(url_for('auth.register'))
+    return render_template('pages/register.html', form=form)
+
+
+@auth.route('/login', methods=['GET', 'POST'])
+def login():
+    form = UserLoginForm(request.form)
+    email = form.email.data
+    try:
```

Comment:
you can put just getting user in try...except and change exception type to `NoResultFound`